### PR TITLE
Mulebot UI refactor

### PR DESCRIPTION
**Pull request by Ben10Omintrix**
## About The Pull Request
refactors mulebot UI into typescript. i also did some very minor layout changes to the UI to better seperate things a bit. this serves as part 1 to the mulebot refactor 
muleui

## Why It's Good For The Game
refactors mulebot UI into typescript

## Changelog
:cl:
refactor: mulebot UI has been refactored
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/Mule.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        <br />
+        <Button.Checkbox checked={autoPickup} onClick={() => act('autopick')}>
+          Auto-Pickup
+        </Button.Checkbox>
+        <br />
+        <Button.Checkbox checked={reportDelivery} onClick={() => act('report')}>
+          Report-Delivery
+        </Button.Checkbox>
+        <br />
+        {allowPossession && (
```
Comment (jlsnow301) [inline tgui/packages/tgui/interfaces/Mule.tsx:100]: booleanlike will show 0 when its off
```suggestion
        {!!allowPossession && (
```

**Comment by deathrobotpunch** [comment]
Mulebots my beloved are finally getting the recognition they deserve

**Comment by Gaxeer** [comment]
can you make bottom buttons take all available space? Will be much more better IMO

**Comment by Ben10Omintrix** [comment]
Sorry I didn’t see the review notifications. I’ll fix the checks tomorrow

**Pull request merged**